mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -349,7 +349,7 @@
         "MOM_Settings_RJ_Enable_Rocket_Decals" "Rocket explosion decals"
         "MOM_Settings_RJ_Enable_Rocket_Decals_TT" "When enabled, rockets will create decals upon explosion."
         "MOM_Settings_RJ_Enable_Center_Fire" "Fire rocket projectiles from screen center"
-        "MOM_Settings_RJ_Enable_Center_Fire_TT" "When enabled, rockets will fire from the center of the screen like the Original from Team Fortress 2 / Quake."
+        "MOM_Settings_RJ_Enable_Center_Fire_TT" "When enabled, rockets will fire from the center of the screen like the Original from Team Fortress 2 / Quake. Cannot be changed during a run."
```

Comment:
I think this could be pulled into a new string `MOM_Settings_Run_Disallow_TT`:
`Note: The timer must not be running for this setting to change.`

And put it on the checkbox, as they have to click the checkbox to change the setting and will likely see this tooltip better there.

Reply:
This allows us to expand it over to any other checkbox / setting that may be limited by not while timer is running.